EtlPipelineBlobTrigger/course_stats.py

```diff
@@ -125,6 +127,43 @@ def get_stats(self, raw_course_data):
         return self.shared_utils.get_json_list(raw_course_data, self.get_key)
 
 
+class JobType:
+    """Extracts and transforms the JobType course element"""
+
+    def __init__(self):
+        self.xml_element_key = 'JOBTYPE'
+        self.xml_subj_key = 'JOBSBJ'
+        self.xml_agg_key = 'JOBAGG'
+        self.xml_unavail_reason_key = 'JOBUNAVAILREASON'
+
+        self.shared_utils = SharedUtils(self.xml_element_key,
+                                        self.xml_subj_key, self.xml_agg_key,
+                                        self.xml_unavail_reason_key)
+
+    def get_key(self, xml_key):
+        return {
+            'JOBUNAVAILREASON':
+            'unavailable',
+            "JOBPOP":
+            'number_of_students',
+            "JOBAGG":
+            'aggregation_level',
+            "JOBSBJ":
+            'subject',
+            'JOBRESP_RATE':
+            'resp_rate',
+            "PROFMAN":
+            'proportion_of_students_in_professional_or_managerial_jobs',
```

Comment:
Sorry, no need to append json variables with `proportion_of_students_in_`

Reply:
No problem. Have updated.